Vagrantfile

```diff
@@ -0,0 +1,81 @@
+# -*- mode: ruby -*-
```

Comment:
After we will merge #6 you will need to rebase your PR 
Then this file probably will be redundant